packages/wrangler/src/__tests__/init.test.ts

```diff
@@ -2077,22 +2079,13 @@ describe("init", () => {
 					binding: "test-bucket",
 				},
 			],
-			namespaces: [
+			dispatch_namespaces: [
 				{
 					binding: "name-namespace-mock",
 					namespace: "namespace-mock",
 				},
 			],
-			routes: [
-				{
-					id: "some-route-id",
-					pattern: "delta.quadrant",
-				},
-				{
-					id: "some-route-id",
-					pattern: "alpha.quadrant",
-				},
-			],
+			routes: ["delta.quadrant", "alpha.quadrant"],
```

Comment:
Is this the up to date syntax? The docs at https://developers.cloudflare.com/workers/wrangler/configuration/ suggest it's:
```ts
route = "routes" | { pattern = "*", zone_id = "ZONE_ID" } | { pattern = "*", zone_name = "ZONE_NAME" }
```

Reply:
Nice catch! Thanks for the callout.